src/groovy/org/pih/warehouse/tablero/ColorNumber.groovy

```diff
@@ -1,23 +1,34 @@
-
 package org.pih.warehouse.tablero
 
 class ColorNumber implements Serializable {
-    
-    Long value;
+
+    def value;
     String subtitle;
     String link;
+    String color;
 
-    ColorNumber(Long value, String subtitle, String link = null) {
+    ColorNumber(def value, String subtitle, String link = null, String color = null) {
         this.value = value;
         this.subtitle = subtitle;
         this.link = link;
+        this.color = color;
+    }
+
+    def setConditionalColors(def errorCondition, def successCondition) {
+        switch (this.value) {
+            case { it >= successCondition }: this.color = 'success'; break;
+            case { it >= errorCondition && it < successCondition }: this.color = 'warning'; break;
+            case { it < errorCondition }: this.color = 'error'; break;
+            default: this.color = null;
```

Comment:
Is it ok that this is null by default instead of the 'default' name? Will it be properly handled by the 'getColorByName'?

Reply:
Can this be null?

Reply:
No, I have changed it by a default color